annif/cli.py

```diff
@@ -56,7 +56,7 @@ def run_list_projects():
 
 
 @cli.command("show-project")
-@click.argument("project_id")
+@click.argument("project_id", shell_complete=cli_util.complete_project_id)
```

Comment:
How about defining a cli_util.project_id decorator instead of repeating this many times?